tools/tests/test_lint.py

```
@@ -1144,3 +1456,313 @@ def test_non_mapping_dispositions_and_staffing_are_findings(tmp_path):
+
+
+def test_cell_frontmatter_fires_when_the_description_is_missing(tmp_path):
+    make_clean_tree(tmp_path)
+    cell = tmp_path / "skills" / "charter" / "SKILL.md"
+    cell.write_text(
+        cell.read_text(encoding="utf-8").replace(
+            "description: The binding rules." + NL, ""),
+        encoding="utf-8",
+    )
+    findings = [f for f in lint.run(tmp_path) if "cell-frontmatter" in f]
+    assert len(findings) == 1 and "no description" in findings[0]
```

Ruff (0.16.2)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)